server/postgresql/seed/dishes_seed.js

```diff
@@ -0,0 +1,24 @@
+const faker = require('faker');
+const fs = require('fs');
+
```

Comment:
continue using es6 syntax through the whole file instead of switching between var and let/const unless you have a specific reason to revert to es5